Sources/ProjectDescription/ResourceFileElements.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+/// Resource file elements
+///
+/// - resources: list of resource file elements
+///
+public struct ResourceFileElements: Codable, Equatable {
+    /// List of resource file elements
+    public let resources: [ResourceFileElement]
```

Comment:
This type is missing a public initializer that takes an array of `ResourceFileElement`, @kwridan. Can we do a bugfix release if I open a PR to add the init? I need this in order to update to 1.35.0

Reply:
A fix already landed
https://github.com/tuist/tuist/pull/2541

should be in the next release :)

Reply:
wow, thanks! I love this project :D

Reply:
I'll create a new release this evening 🚀